### ISSUE TITLE
Media query initiation of Zoom plugin on product page

### ISSUE THREAD

**Issue by curtiswilkinson** [NetoECommerce/Skeletal#90]
The zoom hover effect is not instances outside of mobile devices and can lead to some issues when touch screens register clicks as hover events. 

We can media query the two initiations of this plugin on /private/www/netosuite/Templates/skeletal/products/template.html

Change:

```
$(document).ready(function(){
      $('.zoom').zoom();
});
```

TO:

```
$(document).ready(function(){
	if (window.screen.width > 767) {
		$('.zoom').zoom();
	}
});
```

And change: 

```
'onReady' : function () {
	$('.addtocart').button("reset");
	$('.zoom').zoom();
},
```

To:

```
'onReady' : function () {
	$('.addtocart').button("reset");
	 if (window.screen.width > 767) {
		$('.zoom').zoom();
	}
},
```

**Comment by ryvanvel** [comment]
Didn't implement the changes - might just drop the zoom from default skeletal and make it a tweak.

**Comment by ryvanvel** [comment]
Ended up doing neither of the above, CSS fix added to less file in 54d34c1
